src/lib/layer.js

```diff
@@ -455,6 +456,26 @@ export default class Layer {
     // End lifecycle method
   }
 
+  // Checks state of attributes and model
+  // TODO - is attribute manager needed? - Model should be enough.
+  getNeedsRedraw({clearRedrawFlags = false} = {}) {
```

Comment:
The `clearRedrawFlags` arg always feels like a potential source of bugs... Will it be better if we reset the flag when `drawLayer` is called?